py4DSTEM/process/calibration/origin.py

```diff
@@ -224,6 +230,164 @@ def get_origin_from_braggpeaks(braggpeaks,Q_Nx,Q_Ny,findcenter='CoM',bvm=None):
 
     return qx0, qy0, braggvectormap
 
+
+def get_origin_brightest_disk(
+        datacube,
+        probe_kernel,
+        qxyInit = None,
+        probe_mask_size=None,
+        subpixel=None,
+        upsample_factor=16,
+        mask=None):    
+    """
+    Find the origin for all diffraction patterns in a datacube, by finding the brightest peak and then
+    masking around that peak.
+
+    Args:
+        datacube (DataCube):            the data
+        probe_kernel (array):           probe kernel for disk detection
+        qxyInit (array):                (qx0,qy0) origin for choosing the peak
+        probe_mask_size (float):        mask size in pixels. If set to None, we set it to 2*probe radius estimate.
+        sub_pixel                       'None' or 'poly' or 'multicorr'
+        upsample_factor (int):          upsample factor
+        mask (ndarray or None):         if not None, should be an (Q_Nx,Q_Ny) shaped
+                                        boolean array. Mask will be applied to diffraction
+                                        patterns before finding the center.
+        probe_mask_std_scale (float):   size of Gaussian mask sigma. If set to None, function will estimate probe size.
```

Comment:
in the google style docstrings, there is a single space between the arg and its descriptive string, i.e.
```
Args:
    arg1 (int): this is the first argument
    arg2 (float): this is the second argument
````